mcdowellcv.cls

```diff
@@ -29,30 +29,44 @@
   \def\mainfontface{Calibri}
 }
 
+% Set up papersize according to options, default to US letter
+\def\papersize{letterpaper}
+%Set up bool for adjusting vspacing later
+\newif\ifpapersize
```

Comment:
There are many paper formats than `letterpaper` and `a4paper`. In order to allow for those, we can use some package supporting the `switch` construct, such as `boolexpr`, and then just switch over the `\papersize` wherever needed. (source: http://tex.stackexchange.com/questions/29133/how-to-create-switch-structure-comparing-strings-in-latex)